mauto/mauto.c

```diff
@@ -0,0 +1,30 @@
+#include "mauto.h"
+
+static bool avoiding = false;
+
+/**
+ * @brief Goes forward and turns left in case there is an obstacle.
+ * 
+ * @param cmd 
+ */
+void
+auto_handle(json_command *cmd)
```

Comment:
Is this param necessary?

Reply:
Param is not necessary for now, but not sure about the future implementation. Having the whole command can come in handy later.

Reply:
Ok, then I would mark it as /* currently unused */